doc/1.4/running-dmlc.md

```diff
@@ -110,6 +110,12 @@ This generates a DML debug file leveraged by debug-simics, and
 causes generated C code to follow the DML code more closely.
 </dd><dt>
 
+--coverity
+</dt><dd>
+
+Adds analysis annotations for common false positives in generated code.
```

Comment:
You should mention Coverity in the description ("the coverity static analysis tool" or something). Unsure if you also should include ™ or something; @sgronlun probably knows.

Reply:
Based on [this](https://www.synopsys.com/software-integrity/security-testing/static-analysis-sast.html) and prior experiences of rigid Intel policies, my bet is that it should be expressed `Synopsys® Coverity®`

Reply:
Now doing so